buildSrc/subprojects/build/build.gradle.kts

```diff
@@ -1,9 +1,10 @@
 dependencies {
+    api("com.google.guava:guava:26.0-jre")
+    api("org.asciidoctor:asciidoctor-gradle-plugin:1.5.9.2")
```

Comment:
Why is this now `api`? (it was `implementation ` before)

Reply:
It's now used by another subproject: https://github.com/gradle/gradle/pull/8148/files#diff-70f9b6d41c7b47bc905a4e065e2355b1R106

Reply:
Thanks. I missed that. All good then.